src/platforms/monday/client.ts

```diff
@@ -0,0 +1,28 @@
+import { formatUrl, makeRequestFactory } from '@/sdk/client';
+import { API_VERSION, BASE_URL } from './constants';
+import { mondayQueryResponse } from './schemas';
+
+const request = makeRequestFactory(async (auth, options) => {
+  return {
+    ...options,
+    url: formatUrl(BASE_URL, options.url),
+    headers: {
+      ...options.headers,
+      Authorization: `Bearer ${await auth.getToken()}`,
+    },
+  };
+});
+
+const makeClient = () => {
+  return {
+    query: request(({ query }: { query: string }) => ({
+      url: `/${API_VERSION}`,
+      method: 'POST',
+      schema: mondayQueryResponse,
+      json: { query },
+    })),
+    passthrough: request.passthrough(),
```

Comment:
is the graphql action much easier to use than passthrough? seems like they're basically the same at first glance